test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -102,7 +102,7 @@ public IList<TestDataset> GetDatasetsForClassificationWeightingPredictorsTest()
         /// <summary>
         ///A test for binary classifiers
         ///</summary>
-        [Fact]
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
```

Comment:
I think we should fix/adjust the baseline mechanism so tests like this can be executed. We shouldn't disable tests just because the baseline differs.

Reply:
I agree, but it requires more investigation to determine what differs and why. I saw a number of other tests that are already disabled due to baseline differences as well (for all targets), that also need investigation.

Reply:
Looking at a handful of the differences, they generally seem to be minor differences in the computed result (for example: `1.4157174495771871` vs `1.4157174495771872`). This is likely fallout from alignment differences and algorithm differences for various `System.Math` (and other) functions.

The comparison logic, in general, likely needs an overhaul (to properly account for these differences cross-architecture and cross-platform).

Reply:
CC. @GalOshri, @shauheen, @TomFinley, @Zruty0